ffi/SDL2_0.lua

```diff
@@ -121,6 +121,11 @@ function S.open(w, h, x, y)
     S.renderer = SDL.SDL_CreateRenderer(S.screen, -1, 0)
     S.texture = S.createTexture()
 
+    -- Start delivering Unicode text as well as keypresses - this will
+    -- handle things like Shift-numbers and letters according to the layout
+    -- and will deliver SDL_TEXTINPUT with that text.
+    SDL.SDL_StartTextInput()
```

Comment:
Given that "On some platforms using this function activates the screen keyboard.", this shouldn't be done here but only where it's used.